docs/features/experiment.md

```diff
@@ -243,5 +245,9 @@ to `experiment-baseline`, leaving the remaining 90% of traffic to the old stack.
 
 !!! note
     When a weighted experiment step with traffic routing is used, a
-    service is auto-created for each experiment template. The traffic routers use
-    this service to send traffic to the experiment pods.
\ No newline at end of file
+    Service is auto-created for each experiment template. The traffic routers use
+    this service to send traffic to the experiment pods.
+
+By default, the generated Service has the name of the ReplicaSet and inherits
```

Comment:
needs this https://github.com/argoproj/argo-rollouts/pull/2396/files to be merged first

Reply:
has been merged